ci-operator/step-registry/cucushift/upgrade/helper-service/build-info/cucushift-upgrade-helper-service-build-info-commands.sh

```diff
@@ -12,3 +12,10 @@ if [[ -n "${RELEASE_IMAGE_LATEST}" ]]; then
     LATEST_VERSION="$(oc adm release info "${RELEASE_IMAGE_LATEST}" --output=jsonpath="{.metadata.version}")"
     echo "${LATEST_VERSION}" > "${BUILD_INFO_FILE}"
 fi
+if [[ -n "${RELEASE_IMAGE_TARGET}" ]]; then 
+    echo "${RELEASE_IMAGE_TARGET}"
+    echo "Release info for: ${RELEASE_IMAGE_TARGET}"
+    oc adm release info "${RELEASE_IMAGE_TARGET}" || true
+    TARGET_VERSION="$(oc adm release info "${RELEASE_IMAGE_TARGET}" --output=jsonpath="{.metadata.version}")"
+    echo "${TARGET_VERSION}" > "${BUILD_INFO_FILE}"
```

Comment:
Would this override the `${BUILD_INFO_FILE}` and wipe out the `LATEST_VERSION` information?

Reply:
BUILD_INFO_FILE file will get overwritten with target version build number.

Reply:
We should use `>>` there, as we want to save both the from/to versions.

Reply:
@liangxia ok, let me update it and send new commit

Reply:
fixed in new commit